manifests/init.pp

```diff
@@ -116,63 +122,105 @@
 #             email     => 'mail',
 #           },
 #           group_mappings => [
-#             { group_dn => '50000', org_role => 'Admin'  },
+#             { group_dn => 'admin', org_role => 'Admin'  },
 #             { group_dn => '50001', org_role => 'Editor' },
 #           ],
 #         },
 #       ],
 #     },
 #   }
 #
-# @author Lucas Yamanishi <lucas.yamanishi@onyxpoint.com>
+# @author https://github.com/simp/pupmod-simp-simp_grafana/graphs/contributors
 #
 class simp_grafana (
-  Simplib::Netlist              $trusted_nets            = $::simp_grafana::params::trusted_nets,
-  Boolean                       $firewall                = $::simp_grafana::params::firewall,
+  Hash                          $default_cfg,
```

Comment:
IMHO $app_pki_dir, $install_method, $rpm_iteration, $simp_dashboards,and  $use_internet_repo should be defined here and not in data in modules.  They are simple strings/booleans.